### ISSUE TITLE
HTML images have the src and " stripped off

### ISSUE THREAD

**Issue by jptechnical** [mokeyish/obsidian-enhancing-export#17]
First of all, AWESOME addon, great work!

In my testing I found that images are broken if they are already HTML. 

For example, this in my obsidian markdown file: 
```html
<img align="right" alt="This is boring" width="150" src="https://c.tenor.com/KmPFMGQ07-4AAAAd/hffgf.gif/">
```
becomes this in both the hugo and regular markdown conversions:
```html
<img align="right" alt="This is boring" width="150" https://c.tenor.com/KmPFMGQ07-4AAAAd/hffgf.gif/>
```

Any suggestions?

**Comment by mokeyish** [comment]
@jptechnical Hi, I can't reproduce it.

![图片](https://user-images.githubusercontent.com/16131917/184523961-5cad8872-8278-4a76-8deb-60e67415845c.png)

**Comment by mokeyish** [comment]
Are you installed the latest pandoc?

**Comment by jptechnical** [comment]
I believe so. Here are my specifics and the entire .md file. 

Mac Studio (2022)
Apple M1 Max
32 GB
Monterey 12.5

Obsidian v0.15.9
Obsidian Enhancing Export v1.1.2
no other community plugins

```bash
$  which pandoc
/opt/homebrew/bin/pandoc

$ pandoc -v
pandoc 2.19
Compiled with pandoc-types 1.22.2, texmath 0.12.5.2, skylighting 0.13,
citeproc 0.8.0.1, ipynb 0.2, hslua 2.2.1
Scripting engine: Lua 5.4
User data directory: /Users/jp/.local/share/pandoc
Copyright (C) 2006-2022 John MacFarlane. Web:  https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.
```
[How to Set File Permissions on Windows.md](https://github.com/mokeyish/obsidian-enhancing-export/files/9335066/How.to.Set.File.Permissions.on.Windows.md)

**Comment by jptechnical** [comment]
Note that it is also adding `\` to the `[` characters. 

So the string `# [[How to Set File Permissions on Windows]]`
is exported as `# \[\[How to Set File Permissions on Windows\]\]`

Attached is the file after exporting on my system.
[How to Set File Permissions on Windows.md](https://github.com/mokeyish/obsidian-enhancing-export/files/9335094/How.to.Set.File.Permissions.on.Windows.md)

**Comment by mokeyish** [comment]
Thanks for your feedback, I have reproduce it, I'ill fix it this weekend.

**Comment by mokeyish** [comment]
@jptechnical I have fix it in version 1.1.4.

**Comment by jptechnical** [comment]
Looks good, thanks, good work!
